### DISCUSSION TITLE
Which Italian composer is known for operas including Rigoletto and La Traviata?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#661] [Galaxy Trivia] [answered]
**Bonus:** What was the last opera you went to or plan to go to?

**Comment by peterbaccaro** [comment, marked as answer]
The Italian composer who is known for operas including "Rigoletto" and "La Traviata" is **Giuseppe Verdi**.

I have never been to see an opera.  I was hoping to go to see an opera when I visited Vienna however I went there in the summer and the opera house is closed in July and August.
The closest thing to seeing an opera was when I did a tour of the Hungarian State Opera House in Budapest and took the option of having a short preview of an opera at the end from one of their opera singers.

**Reply by ParanoidUser** [reply]
Great places to go! Thanks for sharing! I'm looking forward to seeing *Tosca* later this year.
